tensorflow/core/kernels/cwise_ops_sycl_common.h

```diff
@@ -154,6 +154,28 @@ struct BinaryFunctor<SYCLDevice, Functor, NDIMS, has_errors> {
   DEFINE_BINARY5(F, T5, T6, T7, T8, T9)
 
 }  // end namespace functor
+
+
+#if defined(__ANDROID_TYPES_SLIM__)
```

Comment:
In the current code, it is needed for cwise ops, see the code at the end of cwise_ops_common.h,  this change is to keep this logic.

As for other sycl kernels, i'll review it when I face them. As for this patch, it just focuses on cwise ops.